core/standalone/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+FROM scala
+ARG OPENWHISK_JAR
+ENV DOCKER_VERSION=18.06.3-ce
+ENV WSK_VERSION=1.0.0
+ADD init /
+ADD stop /bin/stop
+RUN chmod +x /bin/stop ;\
```

Comment:
Just to confirm - For docker-in-docker case is the version of Docker packaged within container needs to be same as one on the host?

Reply:
No. We have this situation for all the openwhisk invokers. They control an external docker that is usually of a different version. I aligned the version of the packaged docker to the same of the invoker image. Strictly speaking is not a "docker-in-docker", is simply the docker client controlling the external docker via a volume mount of the socket. Starting from version 18.x you can use an older docker client to control newer instances (previously you had to specify the API level, no more).